### PR TITLE
id: preserve selectors for "task" and "workflow" properties

### PR DESCRIPTION
**Pull request by oliver-sanders**
* Selectors were not correctly preserved for the `task` and `workflow` properties of `Tokens` objects.
* This effectively stripped cycle/task selectors from IDs passed to `cylc trigger`.
* Closes https://github.com/cylc/cylc-flow/issues/5186

**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Applied any dependency changes to both `setup.cfg` and `conda-environment.yml`.
- [x] Tests are included (or explain why tests are not needed).
- [x] `CHANGES.md` entry included if this is a change that can affect users
- [x] [Cylc-Doc](https://github.com/cylc/cylc-doc) pull request opened if required at cylc/cylc-doc/pull/XXXX.
- [ ] If this is a bug fix, PRs raised to both master and the relevant maintenance branch.

### CODE REVIEW

**Review by hjoliver — APPROVED** [review]
:+1: LGTM

**Review by hjoliver — COMMENTED** [review]
Doesn't fix the original reported problem...

**Review by hjoliver — CHANGES_REQUESTED** [review]
OK, to close the issue we need to detokenise IDs with `selectors=True` in the CLI scripts.  I'll post a PR to your branch.

**Review by MetRonnie — APPROVED** [review]

**Review by MetRonnie — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ Maintenance release.
 [#5192](https://github.com/cylc/cylc-flow/pull/5192) - Recompute runahead limit
 after use of `cylc remove`.
 
+[#5188](https://github.com/cylc/cylc-flow/pull/5188) -
+Fix task state selectors in the `cylc trigger` command.
```
Comment (hjoliver) [inline CHANGES.md:43]:
```suggestion
Fix task state selectors in `cylc trigger` and other commands.
```

**Review by hjoliver — APPROVED** [review]

**Pull request merged**